test/tf_exec.cpp

```diff
@@ -42,6 +43,94 @@ namespace testing {
 
 #define ASSERT_OK(x) ASSERT_EQ((x), ::tensorflow::Status::OK());
 
+Status LoadGraph(const string& graph_file_name,
```

Comment:
Create a new file for multi-threaded tests (optional)

Reply:
Made changes to make this UT run on CPU. But not moving it to a new file for now